### PR TITLE
Kirstie weekly meeting update | 29 July - 12 August

### PR DESCRIPTION
**Pull request by KirstieJane**
Added my "weekly" meeting updates for the last two weeks and updated where.md.

I'm declaring bankrupcy on the weekly meetings from the last few months :sob: I'll try to be better going forwards.

My abstracts & bios file is also not completely up to date, but at I'm moving forward 🚀 🌟 

@sparkler0323, @minnieho1115, @GeorgiaHCA - please review (but don't merge).

### CODE REVIEW

File: Kirstie-Whitaker/abstracts_bios_2019.md [hunk 1]
```diff
@@ -252,34 +256,31 @@ She is the chair of the Turing Institute's Ethics Advisory Group, a Fulbright sc
 Kirstie was named, with her collaborator Petra Vertes, as a [2016 Global Thinker](https://gt.foreignpolicy.com/2016/profile/petra-vertes-and-kirstie-whitaker) by Foreign Policy magazine.
 You can find more information at her lab website: [whitakerlab.github.io](https://whitakerlab.github.io/).
 
-**Slides**: *Figshare link to slides (if applicable)*
+**Slides**: https://doi.org/10.5281/zenodo.3238189
 
 
-### Autistica Discover Conference
+### Talk in OHBM Open Science symposium
```
Comment (minnieho1115) [inline Kirstie-Whitaker/abstracts_bios_2019.md]: Hi,Kirstie.It seems like you deleted somthing here. :sob:

File: Kirstie-Whitaker/abstracts_bios_2020.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+* May
+* June
+* July
+* August
+* September
+* October
+* November
+* December
+
+### Manchester Literary and Philosophical Society
```
Comment (minnieho1115) [inline Kirstie-Whitaker/abstracts_bios_2020.md]: I think it is really interesing talk.But maybe it should be showed in the table content.

File: Kirstie-Whitaker/abstracts_bios_2019.md (hunk 1 above)
Comment (KirstieJane) [reply Kirstie-Whitaker/abstracts_bios_2019.md]: Thanks @minnieho1115! Restoring now!

File: Kirstie-Whitaker/abstracts_bios_2020.md (hunk 2 above)
Comment (KirstieJane) [reply Kirstie-Whitaker/abstracts_bios_2020.md]: Added it! Thank you!

**Comment by KirstieJane** [comment]
Merging as I've opened #292 and it gets confusing to have too many of these open!

**Pull request merged**